opensrp-app/src/main/java/org/smartregister/repository/EventClientRepository.java

```diff
@@ -1233,6 +1272,19 @@ public void markClientValidationStatus(String baseEntityId, boolean valid) {
         }
     }
 
+    public void updateTaskUnprocessedEventStatus(String formSubmissionId) {
```

Comment:
This can be generalized to `updateTaskStatus(String formSubmissionId, String status)`.

Reply:
I have renamed the method `markEventAsTaskUnprocessed` as per the naming convention of that class. The method name is now concise with what it does

Reply:
Cool.